api/accounts.go

```diff
@@ -180,17 +180,29 @@ func (a *API) accountHandler(_ *apirest.APIdata, ctx *httprouter.HTTPContext) er
 //	@Router					/accounts [post]
 func (a *API) accountSetHandler(msg *apirest.APIdata, ctx *httprouter.HTTPContext) error {
 	req := &AccountSet{}
+	if msg == nil {
```

Comment:
you can join both in a single if statement, the second won't be executed if the first fails.